.github/workflows/testing.yml

```diff
@@ -45,7 +45,8 @@ jobs:
           git config --global user.email "github-tracker-bot@taplytics.com"
           git config --global user.name "DevCycle Automation"
           git add ./VERSION.txt
-          git commit -m "Release ${{steps.prepare-release.outputs.next-release-tag}}"
+          git commit -m "Release v${{steps.prepare-release.outputs.next-release-tag}}"
+          cat VERSION.txt
```

Comment:
debug?

Reply:
Yeah, but I felt like it might be handy to keep in.

Reply:
Otherwise you can't see exactly what it put in there from the output.